### PR TITLE
Add missing options, reformat

### PR DESCRIPTION
**Pull request by vuon9**
- Add `verbose` and `quiet` options for start worker command
- Re-format
- Add condition check `$isQuiet` for foreground mode

### CODE REVIEW

**Review by danhunsaker — CHANGES_REQUESTED** [review]
Technical content looks ok, from my cursory glance, but I'm mostly here for the PHP-Resque side of things, rather than the Symfony bundle side. Just one thing of note, to me, with regard to spelling. "Quiet" and "quite" are frequently mixed up, among native and non-native speakers alike, and it appears that has happened, again, here. Other than adjusting that, where it occurs, though, like I said, it looks good to me.

File: Command/StartWorkerCommand.php [hunk 1]
```diff
@@ -18,10 +18,14 @@ protected function configure()
...
+            ->addOption('interval', 'i', InputOption::VALUE_REQUIRED,
+                'How often to check for new jobs across the queues', 5)
             ->addOption('foreground', 'f', InputOption::VALUE_NONE, 'Should the worker run in foreground')
-            ->addOption('memory-limit', 'm', InputOption::VALUE_REQUIRED, 'Force cli memory_limit (expressed in Mbytes)')
-        ;
+            ->addOption('memory-limit', 'm', InputOption::VALUE_REQUIRED,
+                'Force cli memory_limit (expressed in Mbytes)')
+            ->addOption('verbose', 'v', InputOption::VALUE_REQUIRED,
+                'Records more information than the usual logging mode', 1)
+            ->addOption('quite', 'q', InputOption::VALUE_REQUIRED, 'Disable all logs', 0);
```
Comment (danhunsaker) [inline Command/StartWorkerCommand.php]: Should be "quiet", as "quite" is a different word with a different meaning.

File: Command/StartWorkerCommand.php [hunk 2]
```diff
@@ -42,10 +46,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
 
         $env['APP_INCLUDE'] = $this->getContainer()->getParameter('kernel.root_dir').'/bootstrap.php.cache';
-        $env['COUNT']       = $input->getOption('count');
-        $env['INTERVAL']    = $input->getOption('interval');
-        $env['QUEUE']       = $input->getArgument('queues');
-        $env['VERBOSE']     = 1;
+        $env['COUNT'] = $input->getOption('count');
+        $env['INTERVAL'] = $input->getOption('interval');
+        $env['QUEUE'] = $input->getArgument('queues');
+        $env['VERBOSE'] = $input->getOption('verbose');
```
Comment (danhunsaker) [inline Command/StartWorkerCommand.php:52]: Another "quiet" here.

File: Command/StartWorkerCommand.php [hunk 3]
```diff
@@ -56,12 +61,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $env['VVERBOSE'] = 1;
         }
```
Comment (danhunsaker) [inline Command/StartWorkerCommand.php:63]: And again.

File: Command/StartWorkerCommand.php [hunk 4]
```diff
@@ -92,23 +97,23 @@ protected function execute(InputInterface $input, OutputInterface $output)
             '%dir%' => __DIR__.'/../bin',
```
Comment (danhunsaker) [inline Command/StartWorkerCommand.php:97]: Here, too.

File: Command/StartWorkerCommand.php (hunk 2 above)
Comment (vuon9) [reply Command/StartWorkerCommand.php:52]: Fixed! Thanks.

File: Command/StartWorkerCommand.php (hunk 3 above)
Comment (vuon9) [reply Command/StartWorkerCommand.php:63]: Fixed! Thanks.

File: Command/StartWorkerCommand.php (hunk 4 above)
Comment (vuon9) [reply Command/StartWorkerCommand.php:97]: Fixed! Thanks.

**Comment by vuon9** [comment]
@danhunsaker Many thanks for your opinion. I've edited source code and description also. Hope to get comments or merging act from anyone who has permission.

**Review by danhunsaker — APPROVED** [review]
Looks good from my side!

**Pull request closed without merging**